TADashboard/src/main/java/page_objects/DataProfilesDisplayFieldsPage.java

```diff
@@ -0,0 +1,23 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
+import utils.common.Constants;
+
+import java.util.List;
+
+public class DataProfilesDisplayFieldsPage extends BasePage {
+    private Label lblProfileHeader = new Label("css=#profilesettings > tbody > tr > td.profilesettingheader");
+    private Label lblDisplayFields = new Label("css=#profilesettings > tbody label");
+    private CheckBox dynamicDisplayFieldCheckBox = new CheckBox("xpath=//table[@id='profilesettings']/tbody//label[contains(text(),'%s')]/input");
+
+    public List<String> getAllDisplayFieldsLabels() {
+        lblDisplayFields.waitForDisplay();
+        return lblDisplayFields.getListText();
+    }
+
+    public boolean isFieldPrefixedWithCheckBox(String labelName) {
```

Comment:
isDataFieldCheckboxDisplayed
and your argument should mention about datafield instead of label.

Reply:
Yeah